src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -847,10 +847,6 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				if (newConnection.serverCapabilities?.notebookKernelAlias) {
 					this._currentKernelAlias = newConnection.serverCapabilities.notebookKernelAlias;
 					let sqlConnectionProvider = this._kernelDisplayNameToConnectionProviderIds.get('SQL');
```

Comment:
We wouldn't need this line and the line below (850) since the `_kernelDisplayNameToConnectionProviderIds ` for SQL should already be populated and does not need to be set again. 
The only kernel to connection provider map that should be set is for Kusto.